src/modes/overview/vegaSpec.js

```diff
@@ -103,16 +103,21 @@ const stdErrTransformPercent = [
  */
 export function createSpec(sensor, selections, dateRange, valuePatch) {
   const isPercentage = sensor.format === 'percent';
+  const scalePercent = isPercentage ? (v) => v / 100 : (v) => v;
+
+  const clipping = valuePatch && valuePatch.domain ? true : false;
```

Comment:
as far as I understand if `clipping` is false a bunch of layers and calculations are not used. For performance reasons (e.g., in the small multiple case), one could consider removing (or better not adding) them.

Reply:
Yes, it would be good to avoid computing all the extra data and layers if not needed.  That should be doable.